clis/teliod/src/main.rs

```diff
@@ -179,6 +409,9 @@ async fn daemon_event_loop(config: TeliodDaemonConfig) -> Result<(), TeliodError
                 match signal {
                     Some(s @ SIGHUP | s @ SIGTERM | s @ SIGINT | s @ SIGQUIT) => {
                         info!("Received signal {:?}, exiting", Signal::try_from(s));
+                        if let Err(e) = tx.try_send(TelioTaskCmd::Quit) {
+                            error!("Channel closed or telio not yet started. {e}");
+                        };
```

Comment:
This comment sounds incorrect, if Telio is just not started it will be added to the channel's buffer. There is another rare case in which it may fail, though (and I think this is why I think it would be better to use normal `send` here):
```
This method differs from [send](https://docs.rs/tokio/latest/tokio/sync/mpsc/struct.Sender.html#method.send) by returning immediately if the channel’s buffer is full or no receiver is waiting to acquire some data. Compared with [send](https://docs.rs/tokio/latest/tokio/sync/mpsc/struct.Sender.html#method.send), this function has two failure cases instead of one (one for disconnection, one for a full buffer).```

Reply:
Changed to `send` here.